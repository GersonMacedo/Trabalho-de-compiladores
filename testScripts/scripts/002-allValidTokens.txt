! -s LEXICAL -l DEBUG --enable-logs token
! Has all valid tokens, should correctly idenfify all of them
true
false
begin
end
if
then
else
var
:
;
(
)
.
while
do
,
+
-
or
*
/
and
<
>
:=
=
program
integer
boolean
print
