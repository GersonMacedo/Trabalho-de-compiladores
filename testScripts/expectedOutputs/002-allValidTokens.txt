
Performing only lexical analysis at testScripts/scripts/002-allValidTokens.txt
Token::Token(true, 3, 1)
Token::Token(false, 4, 1)
Token::Token(begin, 5, 1)
Token::Token(end, 6, 1)
Token::Token(if, 7, 1)
Token::Token(then, 8, 1)
Token::Token(else, 9, 1)
Token::Token(var, 10, 1)
Token::Token(:, 11, 1)
Token::Token(;, 12, 1)
Token::Token((, 13, 1)
Token::Token(), 14, 1)
Token::Token(., 15, 1)
Token::Token(while, 16, 1)
Token::Token(do, 17, 1)
Token::Token(,, 18, 1)
Token::Token(+, 19, 1)
Token::Token(-, 20, 1)
Token::Token(or, 21, 1)
Token::Token(*, 22, 1)
Token::Token(/, 23, 1)
Token::Token(and, 24, 1)
Token::Token(<, 25, 1)
Token::Token(>, 26, 1)
Token::Token(:=, 27, 1)
Token::Token(=, 28, 1)
Token::Token(program, 29, 1)
Token::Token(integer, 30, 1)
Token::Token(boolean, 31, 1)
Token::Token(print, 32, 1)
Token::Token(<eot>, 33, 1)
No errors found
